solr/core/src/test/org/apache/solr/search/TestRandomCollapseQParserPlugin.java

```diff
@@ -82,7 +82,6 @@ public static void buildIndexAndClient() throws Exception {
 
   @AfterClass
   public static void cleanupStatics() {
```

Comment:
How about marking `NULL_POLICIES` final to make it clearer that it is static but not being cleaned up here because it's never changed in the `BeforeClass` code?